firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java

```diff
@@ -277,12 +289,13 @@ public UpdateTask updateIfNewReleaseAvailable() {
   }
 
   /**
-   * Updates app to the newest release. If the newest release is an APK, downloads the binary and
-   * starts an installation If the newest release is an AAB, directs the tester to the Play app to
-   * complete the download and installation.
+   * 1:24 PM Updates app to the {@link AppDistributionRelease} returned by {@link
+   * #checkForNewRelease}. If the newest release is an APK, downloads the binary and starts an
+   * installation. If the newest release is an AAB, directs the tester to the Play app to complete
+   * the download and installation.
    *
-   * <p>cancels task with FirebaseAppDistributionException with UPDATE_NOT_AVAILABLE exception if no
-   * new release is cached from checkForNewRelease
+   * <p>Cancels task with {@link FirebaseAppDistributionException.Status.UPDATE_NOT_AVAILABLE} if no
```

Comment:
This should be `Status#UPDATE_NOT_AVAILABLE` (and you can drop the FirebaseAppDistributionException).

Reply:
done